src/Azure.Functions.Cli/Actions/DurableActions/DurablePurgeHistory.cs

```diff
@@ -9,6 +9,9 @@ namespace Azure.Functions.Cli.Actions.DurableActions
     [Action(Name = "purge-history", Context = Context.Durable, HelpText = "Purge orchestration instance state, history, and blob storage for orchestrations older than the specified threshold time.")]
     class DurablePurgeHistory : BaseDurableAction
     {
+        public readonly static DateTime CreatedAfterDefault = DateTime.MinValue;
```

Comment:
I think you should add these to DurableManager instead of the DurablePurgeHistory action. Then both DurablePurgeHistory and DurableGetInstances can reference them :)

Reply:
Ah, I hadn't spotted that the values were used in DurableGetInstances as well!